freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -364,13 +364,13 @@ def _record_builds(self, images, image_modules_mapping):
                 )
                 db.session.commit()
 
-                missing_composes = self._missing_composes(
+                compose_source = self._updated_composes(
                     original_odcs_compose_ids,
                     module_name_stream_set,
                     module_name_stream_version_set,
                 )
                 arches = sorted(image["arches"].split())
-                for compose_source in missing_composes:
+                if compose_source:
                     if compose_source in odcs_cache:
                         db_compose = odcs_cache[compose_source]
```

Comment:
Doesn't the `compose_source` need to be sorted before concatenation so it can be used as cache key consistently?

Reply:
Good point. I was tweaking the test codes. Will fix it in the method.